src/pages/ListContainer/ListContainer.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
+import { List } from '../../components/List/List';
+import { UserModal } from '../../components/UserModal/UserModal';
+import { ListTypes } from '../../types/ListTypes';
+import { UserInfo } from '../../types/UserInfo';
+
+import styles from './ListContainer.module.scss';
+import { Filter } from '../../components/Filter/Filter';
+import { FilterFormTypes } from '../../types/FilterFormTypes';
+
+export const ListContainer: React.FC<ListTypes> = ({ list, setList }) => {
+  const [userToEdit, setUserToEdit] = useState<UserInfo | null>(null);
+  const [isModalActive, setIsModalActive] = useState(false);
+  const [firlteredUsers, setFilteredUsers] = useState<UserInfo[]>(list);
+
+  const removeUser = (userObj: UserInfo) => {
+    let newList = [...list];
+    newList.splice(list.indexOf(userObj), 1);
+
+    setList(newList);
+  };
+
+  const handleEdit = (userObj?: UserInfo) => {
+    if (userObj) {
```

Comment:
You don't need to double-check userObject

    if (userObj) {
     setUserToEdit(userObj);
    } else {
      setUserToEdit(null);
    }

Reply:
changed